src/eos-autoupdater.c

```diff
@@ -66,6 +66,7 @@ static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/" PACKAGE "/eos-autoupdater.conf";
 static const char *OLD_CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
 static const char *STATIC_CONFIG_FILE_PATH = PKGDATADIR "/eos-autoupdater.conf";
+static const char *VAR_CONFIG_FILE_PATH = LOCALSTATEDIR "/local/" PACKAGE "/eos-autoupdater.conf";
```

Comment:
@ramcq I am not sure the updater code should be looking in `/usr`. My understanding is that OSTree mounts (or used to mount) contents added to `/usr/local` into `/var/usrlocal` on the deployed system.

In general though, for similar situations we usually add directly files to subdirectories of `/var` from the image builder instead of relying on OSTree implementation details, so I think @pwithnall's approach is correct; I was only suggesting that we avoid `/var/local/$package_name` because that does not seem consistent with the layout of `/var/local`.